lark/visitors.py

```diff
@@ -523,7 +523,7 @@ def v_args(inline: bool = False, meta: bool = False, tree: bool = False, wrapper
 
     Parameters:
         inline (bool, optional): Children are provided as ``*args`` instead of a list argument (not recommended for very long lists).
-        meta (bool, optional): Provides two arguments: ``meta`` and ``children`` (instead of just the latter)
+        meta (bool, optional): Provides two arguments: ``meta`` and ``children`` (instead of just the latter); ``meta`` is only available for external transformers (i.e., those not supplied as ``Lark``'s ``transformer`` parameter).
```

Comment:
What if we phrase it as 

```
``meta`` isn't available for transformers supplied to Lark using the ``transformer`` parameter (aka internal transformers).
```

Reply:
Certainly sounds better/is more direct. I've changed it!